core/types/transaction.go

```diff
@@ -50,8 +50,7 @@ type txdata struct {
 	Price        *big.Int        `json:"gasPrice" gencodec:"required"`
 	GasLimit     uint64          `json:"gas"      gencodec:"required"`
 	ShardID      uint32          `json:"shardID"  gencodec:"required"`
-	ToShardID    uint32          `json:"toShardID" rlp:"nil"` // for cross-shard tx's destination shard ID; nil means intra-shard tx
```

Comment:
why remove ToShardID?  we need it for cross-shard tx.

Reply:
i talked with rj, and synced with richard and michael. we will enable it back when implementing it. i know after the dinner meeting today, we would probably implement the cross shard soon.